op-batcher/batcher/channel_builder.go

```diff
@@ -11,9 +11,9 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
+const FrameV0OverHeadSize = 23
```

Comment:
This should build on top of https://github.com/ethereum-optimism/optimism/pull/5187 & pull in from the derive package.

Reply:
Yep made a note in the pr comment

Reply:
You can rebase this PR on it and set the target branch to it. Then the diff stays clean.